hibernate-osgi/hibernate-osgi.gradle

```diff
@@ -24,18 +24,18 @@ dependencies {
 	testCompile( libraries.shrinkwrap )
 	testCompile( "org.jboss.arquillian.junit:arquillian-junit-container:1.0.3.Final" )
 	testCompile( "org.jboss.osgi.metadata:jbosgi-metadata:3.0.0.CR1" )
-	testRuntime( "org.jboss.arquillian.container:arquillian-osgi-felix:2.0.0.CR4" )
```

Comment:
Try changing this back to CR4 once and see if it works.  If so, I'll need to re-generate hibernate-osgi/src/test/resources/felix-framework.properties-ORIGINAL